test/transform/test_graph_2_simplicial_complex.py

```diff
@@ -0,0 +1,169 @@
+"""Test graph to simplicial complex transformation."""
+
+import unittest
```

Comment:
remove unittest dependency, we're trying to get rid of this and use pytest only.

Reply:
Done!